torchtext/_extension.py

```diff
@@ -1,16 +1,66 @@
-def _init_extension():
-    import importlib
-    import os
+import os
+from pathlib import Path
+
+import torch
+from torchtext._internal import module_utils as _mod_utils  # noqa: F401
```

Comment:
Is this `F401` necessary??

Reply:
You're right it's not since `_mod_utils` is used in the same file.